root_VS2010/programs/C#/10_Build_WebApp_sample.bat

```diff
@@ -13,7 +13,7 @@ set ORG_PATH=%PATH%
 @rem --------------------------------------------------
 @rem Get the path to the executable file.
 @rem --------------------------------------------------
-set CURRENT_DIR=%~dp0
+set CURRENT_DIR="%~dp0"
```

Comment:
@yusukemaegawa 

I modified the "Future merge path".
Please confirm that there is consistency.
https://github.com/OpenTouryoProject/OpenTouryoTemplates/issues/23#issuecomment-171914767